src/utils/estimates.js

```diff
@@ -0,0 +1,43 @@
+import { calculateEstimate } from '@the-collab-lab/shopping-list-utils';
+
+export const getEtimatedNextPurchaseDay = (item) => {
+  //function to convert seconds to days
+  const convertSecondsToDays = (seconds) => {
+    return seconds / 86400;
```

Comment:
what is 86400. MagicNumber usage. You assign this number to const field with descriptive name. Maybe all this const field move to another file (Constants.js)